### PR TITLE
Fix broken URLs

### PR DESCRIPTION
**Pull request by spetrosi**

### CODE REVIEW

File: guides/doc-Administering_Red_Hat_Satellite/topics/compliance-report.adoc [hunk 1]
```diff
@@ -16,7 +16,7 @@ A compliance report consists of the following areas:
 The Evaluation Characteristics area provides details about an evaluation against a specific profile, including the host that was evaluated, the profile used in the evaluation, and when the evaluation started and finished. For reference, the IPv4, IPv6, and MAC addresses of the host are also listed.
 
 
-[options="header",cols="3,6,7"]
+[options="header",cols="3,{ProductVersion}"]
```
Comment (melcorr) [inline guides/doc-Administering_Red_Hat_Satellite/topics/compliance-report.adoc:19]: The dangers of sed :-)

File: guides/doc-Configuring_Load_Balancer/topics/Installing_the_Load_Balancer.adoc [hunk 2]
```diff
@@ -40,7 +40,7 @@ You must configure sticky session on TCP port 443 to request yum metadata for RP
 //| Anaconda | 8000 | TCP | roundrobin | port 8000 on all {SmartProxies}
 | HTTPS | 443 | TCP | source | port 443 on all {SmartProxyServer}s
 | RHSM | 8443 | TCP | roundrobin | port 8443 on all {SmartProxyServer}s
-| AMQP | 5647 | TCP | roundrobin | port 5647 on all {SmartProxyServer}s
+| AMQP | 5{ProductVersion} | TCP | roundrobin | port 5{ProductVersion} on all {SmartProxyServer}s
```
Comment (melcorr) [inline guides/doc-Configuring_Load_Balancer/topics/Installing_the_Load_Balancer.adoc:43]: This has made my morning. 
The most Melanie of sed's performed by spetrosi :)

**Review by melcorr — CHANGES_REQUESTED** [review]

File: guides/doc-Configuring_Load_Balancer/topics/Installing_the_Load_Balancer.adoc (hunk 2 above)
Comment (spetrosi) [reply guides/doc-Configuring_Load_Balancer/topics/Installing_the_Load_Balancer.adoc:43]: Ahahaha oh my, will fix now

**Review by melcorr — APPROVED** [review]
Looks good now!

**Pull request merged**